### PR TITLE
[DOCS] Improve Trusted Apps "wildcard"/"matches" field documentation

### LINKED ISSUE #1467: [DOCS] Improve Trusted Apps "wildcard"/"matches" field documentation
## Description 

We should add extra information to the Trusted Apps section in the documentation regarding "wildcard"/"matches".  

A section similar to the below:

Current behavior is:
- `?` : match any one character
- `*` :  match zero to many characters

## Acceptance Test Criteria

- [ ] Add a section to the existing description of Trusted Apps "wildcard"/"matches" section to include specific character info.
- [ ] `?` : match any one character
- [ ] `*` :  match zero to many characters

### PR DESCRIPTION
**Pull request by joepeeples**
* Resolves #1467.

Preview [here](https://security-docs_1469.docs-preview.app.elstc.co/guide/en/security/master/trusted-apps-ov.html).

### CODE REVIEW

**Comment by kevinlog** [comment]
@ferullo can you also take a look at this?

**Review by ferullo — APPROVED** [review]
👍

**Comment by joepeeples** [comment]
Thanks for the speedy review @kevinlog and @ferullo ! ⚡

**Review by jmikell821 — APPROVED** [review]
LGTM.

File: docs/management/admin/trusted-apps.asciidoc [hunk 1]
```diff
@@ -28,8 +28,8 @@ NOTE: You can only add a single field type value per trusted application. For ex
 +
 
 * `Operator`: Select an operator to define the condition:
-   - `is`: Must be exactly equal to `Value`. This operation is required for the `Hash` and `Signature` field types.   
-   - `matches`: Can include wildcards in `Value`, such as `C:\path\*\app.exe`. This option is only available for the `Path` field type.
+   ** `is`: Must be exactly equal to `Value`. This operation is required for the `Hash` and `Signature` field types.   
+   ** `matches`: Can include wildcards in `Value`, such as `C:\path\*\app.exe`. This option is only available for the `Path` field type. Available wildcards are `?` (match any one character) and `*` (match zero to many characters).
```
Comment (benironside) [inline docs/management/admin/trusted-apps.asciidoc:32]:
```suggestion
   ** `matches`: Can include wildcards in `Value`, such as `C:\path\*\app.exe`. This option is only available for the `Path` field type. Available wildcards are `?` (match one character) and `*` (match zero or more characters).
```

**Review by benironside — APPROVED** [review]
LGTM, left one minor suggestion.

**Pull request merged**